src/packages/quote.py

```diff
@@ -130,28 +147,29 @@ async def remove(self, ctx: commands.Context, *, args=None) -> None:
 			file.truncate()
 
 		if succes:
-			log.info('QUOTE {quote} has been removed')
+			log.info('Quote {quote} has been removed')
 			await ctx.send(f'Quote removed\n> \"{quote["quote"]}\" - {quote["author"]}')
 		else:
-			log.warning(f'QUOTE remove key could not be found in database. Key: {quote_key}')
+			log.warning(f'Quote failed to be removed from database due to unknown key: {quote_key}')
 			await ctx.send(f'Could not find {quote_key} in the database')
 
-	@q.command(aliases=['change'], brief='Edit a quote', description='Either replace a quote completely, only the author, or just the quote.', usage='[author/quote] "quote" - author')  # assuming this server already has a database for them.
+	# Changes a quote in the database
+	@q.command(aliases=['change'], brief='Edit a quote', description='Either replace a quote completely, only the author, or just the quote.', usage='[quote id] (author/quote) "[quote]" - [author]')  # assuming this server already has a database for them.
 	@commands.has_permissions(administrator=True)
 	async def edit(self, ctx: commands.Context, *args) -> None:  # the arg parser can do some weird stuff with quotation marks TODO wdym?!?!
 		try:
 			index = str(int(args[0]))  # check for impostor aka strings
 		except:
-			log.warning(f'QUOTE edit could not find a valid quote_key. Key: {args[0]}')  # we are returning from here.
-			await ctx.send('The requested quote key could not be read')
+			log.warning(f'Quote edit could not find a quote in the database with key: {args[0]}')  # we are returning from here.
+			await ctx.send(f'Could not find {args[0]} in the database')
 			return
 		request, guild_id = args[1], str(ctx.guild.id)
 
 		with open(f'storage/db/quotes/{guild_id}.json', 'r+', encoding='utf-8') as file:  # starting the file lock
 			quotes = json.load(file)
 			if index not in quotes:  # if the quote is not present we still want to be able to edit this specific index, will screw with the max function in q add
 				quotes[index] = {'quote': '', 'author': '', 'remove_votes': [], 'remove_vetos': [], 'id': int(index)}
-			if request == 'author' or request == 'auteur':
```

Comment:
you don't like dutch compatibility?

Reply:
no :)
again this is due to consistency. Maybe fully support dutch one day

Reply:
Yeah that's fair imo

Reply:
aight cool, maybe if someone is bored, I'll put it up as an issue.